kubernetes/create-weblogic-domain-inputs.yaml

```diff
@@ -89,9 +89,16 @@ exposeAdminNodePort: false
 # Name of the domain namespace
 namespace: default
 
-# Load balancer to deploy.  Supported values are:TRAEFIK, NONE
+# Load balancer to deploy.  Supported values are:TRAEFIK, APACHE, NONE
 loadBalancer: TRAEFIK
 
+# Load balancer app prepath 
+loadBalancerAppPrepath: /
```

Comment:
It looks like Prepath is only used by the Apache LB -- if so, the comment above should mention this.  Also, I wonder if the the Prepath and VolumePath fields comments should have a little more detail to help explain their purpose.